src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.java

```diff
@@ -0,0 +1,61 @@
+package me.zeroeightsix.kami.module.modules.combat;
+
+import me.zero.alpine.listener.EventHandler;
+import me.zero.alpine.listener.Listener;
+import me.zeroeightsix.kami.module.Module;
+import me.zeroeightsix.kami.module.ModuleManager;
+import me.zeroeightsix.kami.setting.Setting;
+import me.zeroeightsix.kami.setting.Settings;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraftforge.event.entity.player.AttackEntityEvent;
+
+/**
+ * @author polymer
+ * @author cookiedragon234
+ * Updated by polymer 10 March 2020
+ * Updated by S-B99 on 10/04/20
+ */
+@Module.Info(name = "AutoEZ", category = Module.Category.COMBAT, description = "Sends an insult in chat after killing someone")
+public class AutoEZ extends Module {
+	private Setting<Mode> mode = register(Settings.e("Mode", Mode.ONTOP));
+	EntityPlayer focus;
+	int hasBeenCombat;
+	enum Mode {GG, ONTOP, EZD, EZ_HYPIXEL, NAENAE }
+
+	private String getText(Mode m) {
```

Comment:
This could be represented by this:
```Java
enum Mode {
GG("gg, "),
ONTOP("KAMI BLUE on top! ez "),
EZD("You just got ez'd ")
...

public final String text;
private Mode(String text) {
	this.text = text;
}
}

private String getText(Mode m) {
	return m.text;
}
```